lib/linguist/vendor.yml

```diff
@@ -302,3 +302,6 @@
 
 # Android Google APIs
 - (^|/)\.google_apis/
+
+# Jenkins Pipeline
+- ^Jenkinsfile$
```

Comment:
@jglick - is there a reason this should be considered a `vendored?` file for Linguist - this basically means that it won't contribute to language statistics in the GitHub repository (the coloured bar at the top of the repository page)